app/graphql/types/query_type.rb

```diff
@@ -11,7 +11,13 @@ class QueryType < Types::BaseObject
       argument :order_by, Types::ProviderOrder, required: false
     end
     def providers(filters: nil, order_by: nil)
-      ::Provider.where(where(filters)).order(order(order_by))
+      filters = filters.to_h
+      updated_within_days = filters.delete(:updated_within_days)
+      scope = ::Provider.where(where(filters)).order(order(order_by))
+      if updated_within_days
+        scope = scope.where("updated_at > ?", updated_within_days.days.ago)
```

Comment:
Should this `where` lookup be combined with the lookup on line 16? Or, is the DB only read once to begin with?

Reply:
It can remain here— the DB query won't happen until the scope is finally used outside of this method call.